vars/skipStage.groovy

```diff
@@ -22,15 +22,33 @@ String run_default_skipped_stage(String stage) {
 }
 
 boolean is_pr() {
+    if (params.CI_MORE_FUNCTIONAL_PR_TESTS) {
```

Comment:
https://github.com/daos-stack/daos/blob/a49bf5180d0d3d865acbd9d2ad2bc68331a0431c/Jenkinsfile#L128-L130 is unclear to me as to what that parameter is for, why I'd use it, what I'd put in it.

Which makes me wonder why the parameters are not just mirrors (in groovy name, description, etc.) of commit pragmas.

Reply:
Currently some behavior of the skips are based on if this is a pr branch or not.

This parameter is used to disable the "is_pr() test so that I can use parameters to enable tests that would normally be skipped in a PR.

It is a boolean parameter.   Unfortunately the help text in the jenkinsfile is not displayed for boolean parameters.